parcel_test.go

```diff
@@ -108,14 +161,17 @@ func TestGetByClient(t *testing.T) {
 	}
 
 	// get by client
-	storedParcels, err := // получите список посылок по идентификатору клиента, сохранённого в переменной client
-	// убедитесь в отсутствии ошибки
-	// убедитесь, что количество полученных посылок совпадает с количеством добавленных
+	storedParcels, err := store.GetByClient(client)    // получите список посылок по идентификатору клиента, сохранённого в переменной client
+	require.NoError(t, err)                            // убедитесь в отсутствии ошибки
+	require.Equal(t, len(parcels), len(storedParcels)) // убедитесь, что количество полученных посылок совпадает с количеством добавленных
 
 	// check
 	for _, parcel := range storedParcels {
 		// в parcelMap лежат добавленные посылки, ключ - идентификатор посылки, значение - сама посылка
 		// убедитесь, что все посылки из storedParcels есть в parcelMap
 		// убедитесь, что значения полей полученных посылок заполнены верно
+		p, err := parcelMap[parcel.Number]
```

Comment:
Обычно называют не `err`, так как это bool
```suggestion
		p, ok := parcelMap[parcel.Number]
```
```suggestion
		p, exists := parcelMap[parcel.Number]
```

Reply:
привел несколько распространненых вариантов